src/projectile.cpp

```diff
@@ -108,22 +108,22 @@ void apply_ammo_effects( const tripoint &p, const std::set<std::string> &effects
     if( effects.count( "NAPALM" ) > 0 ) {
         g->explosion( p, 4, 0.7, true );
         // More intense fire near the center
-        for( auto && pt : g->m.points_in_radius( p, 1, 0 ) ) {
+        for( auto &&pt : g->m.points_in_radius( p, 1, 0 ) ) {
```

Comment:
Agreed, when we started allowing c++11 features, a few people went a bit crazy with some of the new features, foremost amoung them were rvalue refrences/move operations and auto.  At some point I'm going to do a series of code audits and make a lot of this stuff more straightforward.